popSimManu.tex

```diff
@@ -834,6 +834,16 @@ \subsection*{Model quality control}
 Further details of our QC process can be found in our developer documentation
 (\url{https://stdpopsim.readthedocs.io/en/latest/development.html#}).
 
+The possibility for error and the importance of careful qualty control
+was inadvertantly illustrated during our own development process:
+during the final revisions of this paper, we noticed that the 
+OutOfAfrica\_3G09 model \citep{gutenkunst2009inferring}
+had not in fact gone through the QC process.
+Then, the QC process revealed that not only was our implementation
+slightly wrong---migration rates had not been set to zero to the European population in the longest-ago time period
```

Comment:
i realize that this already got merged but there this sentence needs a bit of a rewrite-- the first clause ''not only'' doesn't really hook up with anything on the second half of the sentence.

flagging this for edits